From 642b3360c582855a05e88af60e1428468f7c3758 Mon Sep 17 00:00:00 2001
From: Jon TURNEY <jon.turney@dronecode.org.uk>
Date: Thu, 24 Jul 2014 16:50:10 +0100
Subject: [PATCH 15/17] Include _glapi_create_table_from_handle() in shared
 glapi as well

glapi_gentable.c, should be included in shared glapi as well, to provide
_glapi_create_table_from_handle()

Signed-off-by: Jon TURNEY <jon.turney@dronecode.org.uk>
---
 src/mapi/glapi/Makefile.am      | 4 ++--
 src/mapi/glapi/Makefile.sources | 4 +++-
 2 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/src/mapi/glapi/Makefile.am b/src/mapi/glapi/Makefile.am
index e425bac..cadcd69 100644
--- a/src/mapi/glapi/Makefile.am
+++ b/src/mapi/glapi/Makefile.am
@@ -48,13 +48,13 @@ if HAVE_SHARED_GLAPI
 AM_CPPFLAGS += \
 	-DMAPI_MODE_BRIDGE \
 	-DMAPI_ABI_HEADER=\"glapi/glapi_mapi_tmp.h\"
-glapi_GLAPI_sources =
+glapi_GLAPI_sources = $(GLAPI_UTIL_SOURCES)
 glapi_ASM_sources =
 glapi_MAPI_sources = $(MAPI_BRIDGE_FILES)
 else
 AM_CPPFLAGS += \
 	-DMAPI_MODE_UTIL
-glapi_GLAPI_sources = $(GLAPI_SOURCES)
+glapi_GLAPI_sources = $(GLAPI_SOURCES) $(GLAPI_UTIL_SOURCES)
 glapi_ASM_sources = $(GLAPI_ASM_SOURCES)
 glapi_MAPI_sources = $(MAPI_UTIL_FILES)
 endif
diff --git a/src/mapi/glapi/Makefile.sources b/src/mapi/glapi/Makefile.sources
index 04fef6c..aae41a8 100644
--- a/src/mapi/glapi/Makefile.sources
+++ b/src/mapi/glapi/Makefile.sources
@@ -1,9 +1,11 @@
 # src/mapi/glapi/Makefile.sources
 
+GLAPI_UTIL_SOURCES = \
+	glapi_gentable.c
+
 GLAPI_SOURCES = \
 	glapi_dispatch.c \
 	glapi_entrypoint.c \
-	glapi_gentable.c \
 	glapi_getproc.c \
 	glapi_nop.c \
 	glapi.c \
-- 
2.1.0

